pipeline/merger/README.md

```diff
@@ -0,0 +1,25 @@
+# Merger
+
+Merging rules are applied per field, with a matching rule being applied to a single `target` (a Sierra work) and one or more `sources`.
+
+## Items
+
+The works coming into the merger are as follows:
+
+![Works pre-merger](pre-merger-items.png)
```

Comment:
I really like the colour and image, but it makes it hard to review - would a table do?